ramalama/model.py

```diff
@@ -13,8 +14,6 @@ class Model:
 
     def __init__(self, model):
         self.model = model
-        if sys.platform == "darwin":
```

Comment:
It makes more sense to have the --ngl 99 switch here, because below we have now only handled the "run" case, but not the "serve" case

Reply:
Or we can create a new function that both "run" and "serve" call. That would be fine too I guess, because this stuff is irrelevant to pushing/pulling.

Didn't worry about that too much before because:

sys.platform

was a fast and cheap check

Reply:
On macOS, llama.cpp defaults to the Metal backend, so that check doesn't actually do anything in practice.